scripts/base/init-bare.bro

```diff
@@ -3710,11 +3710,36 @@ export {
 	## external harness and shouldn't output anything to the console.
 	const errors_to_stderr = T &redef;
 }
+
+module Fanout;
+
+type Method: enum {
+    METHOD_HASH = 0,
+    METHOD_LB = 1,
+    METHOD_CPU = 2,
+    METHOD_ROLLOVER = 3
+};
+
+type Flag: enum {
+    FLAG_NONE = 0,
+    FLAG_DEFRAG = 0x8000,
+    FLAG_ROLLOVER = 0x1000
+};
+
+export {
+    const enable = F &redef;
+    const id = 0 &redef;
+    const method = METHOD_HASH &redef;
+    const flag = FLAG_NONE &redef;
+}
 module GLOBAL;
 
 ## Number of bytes per packet to capture from live interfaces.
```

Comment:
-	pd = pcap_open_live(props.path.c_str(), SnapLen(), 1, 1, tmp_errbuf);
rsmmr added a note
7 days ago
Can you elaborate why you replaced the pcap_open_live call with the individual calls below? Does this raise the version of libpcap required?
knielander added a note
4 days ago
It doesn't raise the version, but it will allow you to specify the ring buffer size.